src/main/java/me/zeroeightsix/kami/gui/hudgui/elements/client/ModuleList.kt

```diff
@@ -25,27 +25,25 @@ import kotlin.collections.HashMap
 import kotlin.math.max
 
 object ModuleList : HudElement(
-    name = "ModuleList",
     category = Category.CLIENT,
-    description = "List of enabled modules",
     enabledByDefault = true
 ) {
 
-    private val sortingMode by setting("SortingMode", SortingMode.LENGTH)
-    private val showInvisible by setting("ShowInvisible", false)
-    private val rainbow = setting("Rainbow", true)
-    private val rainbowLength = setting("RainbowLength", 10.0f, 1.0f..20.0f, 0.5f, { rainbow.value })
-    private val indexedHue = setting("IndexedHue", 0.5f, 0.0f..1.0f, 0.05f)
-    private val primary = setting("PrimaryColor", ColorHolder(155, 144, 255), false)
-    private val secondary = setting("SecondaryColor", ColorHolder(255, 255, 255), false)
+    private val sortingMode by setting(getTranslationKey("SortingMode"), SortingMode.LENGTH)
+    private val showInvisible by setting(getTranslationKey("ShowInvisible"), false)
+    private val rainbow = setting(getTranslationKey("Rainbow"), true)
+    private val rainbowLength = setting(getTranslationKey("RainbowLength"), 10.0f, 1.0f..20.0f, 0.5f, { rainbow.value })
+    private val indexedHue = setting(getTranslationKey("IndexedHue"), 0.5f, 0.0f..1.0f, 0.05f)
+    private val primary = setting(getTranslationKey("PrimaryColor"), ColorHolder(155, 144, 255), false)
+    private val secondary = setting(getTranslationKey("SecondaryColor"), ColorHolder(255, 255, 255), false)
```

Comment:
Same comment as above, it's too long. I'm not sure why you have multiple methods for the same thing, as well

Reply:
What do you mean by "multiple methods for the same thing" I am not meant to, I might have accidentally duplicated a method, but please can you tell me where then. Is there something that you want me to change about it, how would I go around shortening it, put it back to the previous way and assume that the text coming in is the key?

Reply:
I mean, why do you have `getTranslationKey` *and* `TranslationKeySetText` *and* `TranslationKey`? I don't see a reason for all of these

Reply:
getTranslationKey() automatically adds the prefix, for example if you wanted Rainbow in ModuleList it gets the translation key for gui.hudgui.elements.client.ModuleList.Rainbow. TranslationKeySetText was more for testing purposes and just returns a translation key that has text that will always be returned and doesn't actually attempt to get text for the key, this just made testing something easier and I haven't got around to removing it yet, I need to do that.